### ISSUE TITLE
STDOUT/STDERR redirects and logging

### ISSUE THREAD

**Issue by PinGwynn** [reddec/monexec#8]
Hello,

I've tried to use monexec instead of supservisord, but saw absence of same critical features:

1. Where I can get logs ? When I start the application (in my case it was redis), I see just:
```
[supervisor] 2018/05/23 06:53:45 Runnable charger-phase registered with id d11cf976-4559-415c-841f-6b19aa4b4e00
[supervisor] 2018/05/23 06:53:45 Runnable charger-phase spawned with id d11cf976-4559-415c-841f-6b19aa4b4e00
[supervisor] 2018/05/23 06:53:45 Runnable charger-phase with id d11cf976-4559-415c-841f-6b19aa4b4e00 stopped
[supervisor] 2018/05/23 06:53:45 Runnable with id d11cf976-4559-415c-841f-6b19aa4b4e00 removed
[supervisor] 2018/05/23 06:53:45 Monitor will respawn charger-phase after 5s
[supervisor] 2018/05/23 06:53:50 Runnable charger-phase registered with id 467dc0f1-6167-4f4f-8b5b-360796e797e7
[supervisor] 2018/05/23 06:53:50 Runnable charger-phase spawned with id 467dc0f1-6167-4f4f-8b5b-360796e797e7
[supervisor] 2018/05/23 06:53:50 Runnable charger-phase with id 467dc0f1-6167-4f4f-8b5b-360796e797e7 stopped
[supervisor] 2018/05/23 06:53:50 Runnable with id 467dc0f1-6167-4f4f-8b5b-360796e797e7 removed
[supervisor] 2018/05/23 06:53:50 Monitor will respawn charger-phase after 5s
```
which means that something wrong (I've misspelled run command), but I don't see what exactly wrong

2. How I can redirect stdout/err to file, like I can do with supervisord ?

**Comment by reddec** [comment]
Good day and thanks for good request! 
Monexec can't yet make logging to file (it was not required before). 
I will try to add this feature ASAP.

**Comment by reddec** [comment]
fix non-printable error: e17c80b4858509f24385761fcdcb06af0836a8c5

**Comment by PinGwynn** [comment]
Thank you. I will test it in evening.

**Comment by reddec** [comment]
released in v0.1.5 for every platform. Please re-open issue if you will have any problem with it
